int32 trackingState
geometry_msgs/Point position
int32 jointType
